apps/backoffice-v2/src/domains/filters/fetchers.ts

```diff
@@ -1,14 +1,24 @@
-import { apiClient } from '../../common/api-client/api-client';
 import { z } from 'zod';
-import { handleZodError } from '../../common/utils/handle-zod-error/handle-zod-error';
+import { apiClient } from '../../common/api-client/api-client';
 import { Method } from '../../common/enums';
+import { handleZodError } from '../../common/utils/handle-zod-error/handle-zod-error';
 import { ObjectWithIdSchema } from '../../lib/zod/utils/object-with-id/object-with-id';
 
+export const BaseFilter = ObjectWithIdSchema.extend({
```

Comment:
Keep schema naming convention.
```suggestion
export const BaseFilterSchema = ObjectWithIdSchema.extend({
```

Reply:
The `Base` prefix is only needed for schemas that include self-references as `z.lazy` tends to break types. So you want to have a "base" with no relationships and a non-base for the more complicated and "real" schema.